arches/app/media/js/utils/create-vue-application.js

```diff
@@ -40,8 +40,19 @@ export default async function createVueApplication(vueComponent, themeConfigurat
         });
 
         const app = createApp(vueComponent);
+        const darkModeClass = themeConfiguration.theme.options.darkModeSelector.substring(1);
+        const darkModeStorageKey = `arches.${darkModeClass}`;
```

Comment:
This will create a local storage key `arches.arches-dark`, which is good in case we end up with `arches.my-not-namespaced-dark-selector`?